lib/Mojolicious/Static.pm

```diff
@@ -39,11 +39,16 @@ sub dispatch {
 }
 
 sub file {
-  my ($self, $rel) = @_;
+  my ($self, $rel, $c) = @_;
 
   # Search all paths
   for my $path (@{$self->paths}) {
-    next unless my $asset = $self->_get_file(catfile $path, split('/', $rel));
+    my $asset;
+    unless( $asset = $self->_get_file(catfile $path, split('/', $rel)) ) {
+      $c->app->log->debug("Did not find $rel in $path") if $c;
```

Comment:
This could potentially result in **a lot** of log messages if you have a few plugins loaded that ship with bundled assets.